packages/create-burner/src/manager/burnerManager.ts

```diff
@@ -117,6 +120,10 @@ export class BurnerManager {
     }
 
     public async init(): Promise<void> {
+        if (this.isInitialized) {
+            throw new Error("BurnerManager is already initialized");
```

Comment:
does this need to be an error? maybe a warning is better

Reply:
I used an error to prevent the odd issue where calling `init` twice in a row ends up removing the burner we just created. This allows us to neatly encapsulate both `init` and `create` within a single try-catch block:

```
try {
    await burnerManager.init();
    if (burnerManager.list().length === 0) {
        await burnerManager.create();
    }
} catch (e) {
    console.error(e);
}
```

I prefer this this option but lmk if you prefer the warning and I'll change it